control.tex

```diff
@@ -1,38 +1,33 @@
 \chapter{Encoder Control} \label{encoderControl}
 
-This chapter describes the fields required to control the Trace Encoder.  
+The fields required to control a Trace Encoder are defined in the 
+\href{https://github.com/riscv-non-isa/tg-nexus-trace/blob/master/docs/RISC-V-Trace-Control-Interface.adoc}{RISC-V Trace Control Interface Specification},
+which is intended to apply to any and all RISC-V trace encoders, regardless of encoding protocol.
+This chapter details which of those fields apply to E-Trace.  To avoid replication, descriptions are not provided here; 
+additional E-Trace specific context or clarification is provided only where required.
+
 How fields are organized and accessed (e.g packet based or memory mapped) is outside the scope of this document.  
-A recommended register map can be found at \href{https://github.com/riscv-non-isa/tg-nexus-trace/blob/master/docs/RISC-V-Trace-Control-Interface.adoc\#register-map}
-{https://github.com/riscv-non-isa/tg-nexus-trace/blob/master/docs/RISC-V-Trace-Control-Interface.adoc\#register-map}.
+If a memory mapped approach is adopted, \href{https://github.com/riscv-non-isa/tg-nexus-trace/blob/master/docs/RISC-V-Trace-Control-Interface.adoc\#register-map}
+{this register map} from the RISC-V Trace Control Interface Specification should be used.

```

Comment:
Is there an option to not use memory mapped approach?

Reply:
Yes.  The Siemens Embedded Analytics (formerly UltraSoC) encoder and other debug/diagnostic IP uses a symmetric packet-based communication protocol for both output of trace and input of configuration data.  The configuration data takes the form of packets with payloads up to 31 bytes in length.  There is no bus subordinate interface and no register map.